__tests__/app.test.js

```diff
@@ -0,0 +1,31 @@
+const request = require("supertest");
+const app = require("../db/app.js");
+const seed = require("../db/seeds/seed.js");
+const testData = require("../db/data/test-data/index.js");
+const connection = require("../db/connection.js");
+
+beforeAll(() => {
+	return seed(testData);
+});
```

Comment:
This will work for now, but you will run into issues when you get to a post request. `beforeEach` will serve you better 🙂